HW4/task_1-3.py

```diff
@@ -0,0 +1,14 @@
+
+x = input("Введіть довільне слово або число (ціле): ")
+if (x.isdigit()):
```

Comment:
nit: brackets could be omitted
```suggestion
if x.isdigit():
```